core/src/main/scala/kafka/admin/LogDirsCommand.scala

```diff
@@ -95,6 +99,10 @@ object LogDirsCommand {
           .withRequiredArg
           .describedAs("The server(s) to use for bootstrapping")
```

Comment:
@lindong28 the newly introduced config file might contain `bootstrap.servers=...`. Would it make sense to document that `--bootstrap-server` takes precedence? 
Have you thought of making `--bootstrap-server` optional and read servers from the config file? I know we are not doing this in other tools either, but I'm curious about your opinion.

Reply:
@andrasbeni Thanks for the suggestion. Yes it makes sense to document that `--bootstrap-server` takes precedence. 

Regarding making `--bootstrap-server ` optional, I think this approach also works just like the current solution which keeps `--bootstrap-server` required and `--bootstrap-server` take precedence. The former approach allows user the optional to only specify bootstrap server in the properties file whereas the latter doesn't. Sometimes flexibility is good but sometimes having multiple ways to do the same thing can also be confusing. Since I don't see a clear benefit for the extra flexibility, I would rather choose the easy way not to change whether `--bootstrap-server` is required. If anyone has a good reason to make it optional, I am OK with that as well.